common/src/buffers/in_mem/send.rs

```diff
@@ -81,6 +81,10 @@ impl SendingBuffer for InMemorySendingBuffer {
         }
 
         if available_bytes > 0 {
+            denim_chunks
+                .last_mut()
+                .expect("Should have at least one chunk")
```

Comment:
do not expect do match instead and log or something

Reply:
please do chek on  min_payload_length when making the buffer and error if it fails